src/ui/component-repository/components/list/HubRegistrySwitcher.js

```diff
@@ -33,6 +34,11 @@ const HubRegistrySwitcher = () => {
     dispatch(setInfo({ type: 'Registry' }));
   }, [dispatch]);
 
+  const handleEditRegistry = useCallback((registry) => {
+    dispatch(setVisibleModal(EDIT_REGISTRY_MODAL_ID));
+    dispatch(setInfo({ editData: registry }));
+  }, [dispatch]);
```

Comment:
you don't need `useCallback` in this case, see [here](https://react.dev/reference/react/useCallback#should-you-add-usecallback-everywhere)